combinations/hash.tsv

```diff
@@ -143,3 +143,4 @@ python=3.9,pandas=1.3.0,seaborn=0.11.0
 bwa=0.7.17,samtools=1.9,bcftools=1.9	quay.io/bioconda/base-glibc-debian-bash:latest	0
 star=2.7.9a,samtools=1.13,gawk=5.1.0
 python=3.9,pandas=1.3.0,seaborn=0.11.0,rich=10.6.0,typer=0.3.2,bcbio-gff=0.6.6,dna_features_viewer=3.0.3
+delly=0.7.8,manta=1.4.0,gridss=1.9.0,lumpy-sv=0.2.12,svtyper=0.0.2,survivor=1.0.6
```

Comment:
@melsiddieg can you use instead many single containers and chain them together in a workflow?

We can not create 1000 of containers with an arbritrary mix of tools, the idea is to create small containers and orchestrate them via workflows.

Reply:
Hi @bgruening 
I agree, that what we are using right now! However, the SV calling is a logical collection and will simplify a lot of pipelines if packaged together in a single handy container. SV calling and postproccessing is generally underserved, and it is a real need if feasible.

Reply:
My point is more that we have a delly container, a manta one and for lumpy and also svtyper. We are wasting a lot of space if we create a seperate container with all of them together, instead of using the smaller single containers. its also more practical for tool updates. Just imagine there is a new delly version, you would need to update this super big continer here and later the same when mata got an update. Instead we prefer to update the smaller containers and keep it seperate.

Does that make sense?